### DISCUSSION TITLE
Consider porting to Zed ?

### DISCUSSION THREAD

**Discussion by fBedecarrats** [brutusyhy/polars-explorer#18] [General]
Hi, I am a researcher working a lot on R and more ocasionnaly on python and Julia. I found out about [Zed](https://zed.dev/) and found it great, but it is lacking some functionality for data science use case. One of them is a data explorer, so I looked for similar projects in Rust on github and found yours. I am not able to judge if it would be easy to port to Rust as an extention. What do you think?

**Comment by brutusyhy** [comment]
Hi! Thank you for your interest in this project and this sounds like an interesting suggestion! Right now I'm not sure how feasible it is and I currently don't have much time to spend on this project, but I will consider it as soon as I have some time!
